<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="gen_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="gen_label">
           <property name="text">
            <string>Encryption Level</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="gen_level_choose">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>RSA-768</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RSA-1024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RSA-2048</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gen_btn">
           <property name="text">
            <string>Generate Key</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="public_label">
        <property name="text">
         <string>Public Key</string>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_1" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QWidget" name="public_widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="n_label">
              <property name="text">
               <string>N</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="n_textEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="e_label">
              <property name="text">
               <string>E</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="e_textEdit"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="private_label">
        <property name="text">
         <string>Private Key</string>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QWidget" name="private_widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="p_label">
              <property name="text">
               <string>P</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="p_textEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="q_label">
              <property name="text">
               <string>Q</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="q_textEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="d_label">
              <property name="text">
               <string>D</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="d_textEdit"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="message_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="message_label">
           <property name="text">
            <string>Message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="m_textEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cipher_label">
           <property name="text">
            <string>Cipher</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="c_textEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="decrypted_label">
           <property name="text">
            <string>Decrypted Message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="dec_textEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="process_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="encrypt_btn">
           <property name="text">
            <string>Encrypt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="decrypt_btn">
           <property name="text">
            <string>Decrypt</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
